terraform/modules/keycloak/output.tf

```diff
@@ -0,0 +1,7 @@
+output "instance_id" {
+  value = aws_instance.nat_instance.id
+}
+
+output "private_ip" {
+  value = aws_instance.nat_instance.private_ip
```

Comment:
Why the IP is IP of NAT instance?

Reply:
đã thay đổi và cập nhật lại file output.tf